docs/getting-started/security-ui.asciidoc

```diff
@@ -91,6 +91,14 @@ The Alerts page allows you to view and manage all alerts to monitor activity wit
 [role="screenshot"]
 image::detections/images/alert-page.png[]
 
+[float]
+=== Findings page
+
+The Findings page provides an overview of your Kubernetes infrastructure configuration's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
```

Comment:
"Kubernetes infrastructure configuration's performance" is a long construction; could we shorten, with "configuration" implied as part of the infrastructure?
```suggestion
The Findings page provides an overview of your Kubernetes infrastructure's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
```